app/controllers/users_controller.rb

```diff
@@ -50,7 +50,7 @@ def set_user
   end
 
   def user_params
-    params.expect(user: [ :email, :password, :is_admin ])
+    params.require(:user).permit(:email, :password, :is_admin, provider_ids: [])
```

Comment:
`expect` is the new way for Rails to manage attributes in protected manner. You can add new parameter to existing flow